setup.py

```diff
@@ -18,6 +18,16 @@ def dependencies(file):
     packages=find_packages(exclude=("tests", "examples")),
     version="0.0.30",
     license="MIT",
+    classifiers=[
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 3.4",
```

Comment:
We don't test the package on Python 3.4. Are you sure it works for this version?

Reply:
Solved, removed it.